### DISCUSSION TITLE
Does PHP 7.4 fit Eventum PHP requirement to 7.3?

### DISCUSSION THREAD

**Discussion by vladsf** [eventum/eventum#1196] [Q&A]
Is Eventum requirement to PHP 7.3 strict? I mean is it 7.3 or 7.3+? Does 7.4 fit the requirement?

**Comment by glensc** [comment]
I'm running on 7.4.14. Where did you read the version info?

**Reply by vladsf** [reply]
I read it from 3.10.0 requirements - https://github.com/eventum/eventum/pull/1012
I am looking into upgrading from 3.9 (PHP 7.2.34) to 3.10.

**Reply by glensc** [reply]
You should be able to figure out version from the upgrade doc:
- https://github.com/eventum/eventum/blob/v3.10.6/docs/wiki/Upgrading.md

send a pr if it's not clear there.

in any case, #1012 bumps the minimum version, the upper bound is likely 7.4, I'm sure 8.0/8.1 will give tons of warnings, and most likely you'll get stuck with composer dependencies.

**Reply by vladsf** [reply]
OK. I will try with 7.4
